src/hooks/useLiveSpecs.ts

```diff
@@ -45,64 +44,49 @@ function useLiveSpecs(specType?: Entity, matchName?: string) {
     };
 }
 
-const LIVE_SPECS_DETAILS_QUERY = `
-                updated_at,
-                created_at,
-                connectorName:connector_title->>en-US::text,
-                connector_tag_documentation_url,
-                writes_to,
-                reads_from,
-                spec
-            `;
+export interface LiveSpecsQuery_details extends LiveSpecsQuery {
+    'id': string;
+    'spec': {
+        schema: {
+            properties: Record<string, any>;
+            required?: string[];
+        };
+        key: string[];
+    };
+    'writes_to': string[] | null;
+    'created_at': string;
+    'connectorName:connector_title->>en-US::text': string;
+    'connector_tag_documentation_url': string;
+    'reads_from': string[] | null;
+}
```

Comment:
Created a proper type that includes everything we're querying for instead of just using the `Schema` fallback to allow us to fetch anything off this interface.